Building on v1.5.0-alpha-2 fails on Windows

I've been trying to use `v1.5.0-alpha-2` and compilation of any Kotlin code fails with:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at io.bazel.kotlin.builder.KotlinBuilderMain.main(KotlinBuilderMain.java:26)
Caused by: java.lang.IllegalStateException: runfile manifest C:/users/odrac/_bazel_odrac/vpohbkhb/execroot/__main__/bazel-out/host/bin/external/io_bazel_rules_kotlin/src/main/kotlin/builder.exe.runfiles/MANIFEST did not contain path mapping for external/com_github_jetbrains_kotlin/lib/jvm-abi-gen.jar
```

I've attached a simple repro project (BUILD, WORKSPACE, and Main.kt files), which look like this:

`WORKSPACE`

```
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

RULES_KOTLIN_VERSION = "v1.5.0-alpha-2"
RULES_KOTLIN_SHA = "6194a864280e1989b6d8118a4aee03bb50edeeae4076e5bc30eef8a98dcd4f07"

http_archive(
    name = "io_bazel_rules_kotlin",
    urls = ["https://github.com/bazelbuild/rules_kotlin/releases/download/%s/rules_kotlin_release.tgz" % RULES_KOTLIN_VERSION],
    sha256 = RULES_KOTLIN_SHA,
)

load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_register_toolchains")
kotlin_repositories()
kt_register_toolchains()
```

`BUILD`

```
load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library", "kt_jvm_binary")

kt_jvm_binary(
    name = "main",
    srcs = ["Main.kt"],
    main_class = "foo.bar.MainKt",
    deps = [
    ]
)
```

`Main.kt`

```
package foo.bar

fun main() {
    println("Hello world")
}
```

Running `bazel build //:main` or `bazel run //:main` fails with the aforementioned exception (I've also attached the full stack trace since the error is a bit larger). 

Note that this also reproduces with `v1.5.0-alpha-3`.

[BugReproProject.zip](https://github.com/bazelbuild/rules_kotlin/files/6017106/BugReproProject.zip)
[Stacktrace.txt](https://github.com/bazelbuild/rules_kotlin/files/6017110/Stacktrace.txt)

--- a/src/main/kotlin/io/bazel/kotlin/builder/cmd/BUILD.bazel
+++ b/src/main/kotlin/io/bazel/kotlin/builder/cmd/BUILD.bazel
@@ -20,6 +20,7 @@ kt_bootstrap_binary(
         "//src/main/kotlin:skip-code-gen",
         "//src/main/kotlin/io/bazel/kotlin/compiler",
         "@com_github_jetbrains_kotlin//:lib/kotlin-compiler.jar",
+        "@com_github_jetbrains_kotlin//:lib/jvm-abi-gen.jar",
     ],
     main_class = "io.bazel.kotlin.builder.cmd.Build",
     runtime_library = ":build_lib",

